File: Sources/eclipse/plugins/com.thalesgroup.orchestra.doors.framework.environment/src/com/thalesgroup/orchestra/doors/framework/environment/model/DoorsFolder.java
/**
 * Copyright (c) THALES, 2011. All rights reserved.
 */
package com.thalesgroup.orchestra.doors.framework.environment.model;

import org.eclipse.swt.graphics.Image;

import com.thalesgroup.orchestra.doors.framework.environment.DoorsActivator;

/**
 * @author T0052089
 */
public class DoorsFolder extends AbstractDoorsContainer {

  public DoorsFolder(AbstractDoorsContainer parent_p, String doorsId_p, String doorsName_p, boolean isVirtual_p) {
    super(parent_p, doorsId_p, doorsName_p, isVirtual_p);
  }

  /**
   * @see com.thalesgroup.orchestra.doors.framework.environment.model.DefaultModel#getImage()
   */
  @Override
  public Image getImage() {
    return DoorsActivator.getInstance().getImage("doors_folder.gif"); //$NON-NLS-1$
  }
}

File: Sources/DoorsEnvironment/Model/DoorsFolder.cs
// ----------------------------------------------------------------------------------------------------
// File Name: DoorsFolder.cs
// Project: DoorsEnvironment
// Copyright (c) Thales, 2010 - 2012. All rights reserved.
// ----------------------------------------------------------------------------------------------------

namespace Orchestra.Framework.Environment.Doors
{
    using System.Collections.Generic;
    using Orchestra.Framework.Core;

    internal class DoorsFolder : DoorsItem
    {
        public List<Artefact> Artefacts { get; set; }
    }
}